DEVELOPMENT.md

```diff
@@ -100,15 +97,21 @@ Currently you must install the Tekton dashboard into the same namespace you wish
 
 ## Iterating
 
-While iterating on the project, you may need to:
+While iterating on the project, the following are common:
```

Comment:
The following what? Operations? Actions?

Reply:
Will revert.